monitoring/metric_test.go

```diff
@@ -0,0 +1,66 @@
+package monitoring_test
+
+import (
+	"time"
+
+	"code.cloudfoundry.org/lager"
+	. "github.com/cloudfoundry-community/splunk-firehose-nozzle/monitoring"
+	"github.com/cloudfoundry-community/splunk-firehose-nozzle/testing"
+	"github.com/cloudfoundry-community/splunk-firehose-nozzle/utils"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Monitoring", func() {
+	var (
+		filtermetric string = "nozzle.queue.percentage,splunk.events.dropped.count,splunk.events.sent.count,firehose.events.dropped.count,firehose.events.received.count,splunk.events.throughput,nozzle.usage.ram,nozzle.usage.cpu,firehose.events.received.count.normalreceive,nozzle.cachehit.memory,nozzle.cachemiss.memory,nozzle.cachehit.remote,nozzle.cachemiss.remote,nozzle.cachehit.boltdb,nozzle.cachemiss.boltdb"
```

Comment:
Why all metrics needed here? You can provide any value. For example: `a,b,c`. Use simple value.

Reply:
Updated in the latest commit